Duvet/SheetViewController.swift

```diff
@@ -193,7 +193,8 @@ public class SheetViewController: UIViewController {
     /// - Parameter notification: The keyboard notification.
     ///
     @objc private func adjustViewForKeyboard(notification: Notification) {
-        guard let userInfo = notification.userInfo,
+        guard sheetView?.configuration.usesDefaultKeyboardAvoidance ?? false,
```

Comment:
What do you think about using `== true` instead of `?? false`? I find it easier to read, but it's super minor.

Reply:
Or alternatively, since I'm looking at this code again... this method really shouldn't run if `sheetView` is nil, since it's only taking action on `sheetView`, so maybe I should just add a `let sheetView = sheetView` in the `guard`'s checks above this one, which would negate the need for a default value. What'dya think?

Reply:
It'll all shake out the same. 🤷 If `sheetView` is nil you'll hit the `?? false` in the current code, or it won't `== true` in my suggestion.

Reply:
Certainly wouldn't hurt to add the `sheetView = sheetView` check. I don't have a preference there.